PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+Addresses Issue # 
```

Comment:
`Addresses issue #x` should be under `## What this does`.

Also, under `## How to test`, add some dummy steps, eg:

1. Go to {link}
2. Do {thing}
3. Observe {reaction}

Otherwise: 👍